### PR TITLE
fix extras

### PR DESCRIPTION
**Pull request by Sampuke**
This just fixes turing on and of vehicle extras!

### CODE REVIEW

**Comment by BerkieBb** [comment]
The third parameter for SetVehicleExtra is a boolean, sending an integer is not really bad as it has compatibility for it but it's not the right practice, simply changing it to `not checked` would do the same.

File: client/menus/vehicles/options.lua [hunk 1]
```diff
@@ -2108,8 +2108,7 @@ lib.registerMenu({
             lib.hideMenu(true)
             return
         end
-
-        SetVehicleExtra(cache.vehicle, args, checked)
+        SetVehicleExtra(cache.vehicle, args.extra, checked and 0 or 1)
```
Comment (BerkieBb) [inline client/menus/vehicles/options.lua:2111]:
```suggestion
        SetVehicleExtra(cache.vehicle, args.extra, not checked)
```
You can add the change easily using this

**Review by BerkieBb — CHANGES_REQUESTED** [review]

File: client/menus/vehicles/options.lua (hunk 1 above)
Comment (Sampuke) [reply client/menus/vehicles/options.lua:2111]: That's wired i have been making some stuff using extras and in the docs it said that it isn't a boolean.

Comment (BerkieBb) [reply client/menus/vehicles/options.lua:2111]: image
https://docs.fivem.net/natives/?_0x7EE3A3C5E4A40CC9

**Pull request merged**